fluidreview/api_test.py

```diff
@@ -200,6 +201,59 @@ def test_process_user_both_exist_with_fluid_id(mocker):
     mock_save_profile.assert_not_called()
 
 
+@pytest.mark.parametrize(['amount_to_pay', 'award_cost', 'sends_email'], [
+    ['', '1000.10', False],
+    ['25.99', '', False],
+    ['', '', True]
+])
+def test_parse_webhook_user(mocker, amount_to_pay, award_cost, sends_email):
+    """Test creation of new Bootcamp if no matching award_id"""
+    user_id = 94379385
+    award_id = 78456
+    award_name = "Best monkey prize"
+    ProfileFactory(fluidreview_id=user_id)
+    mock_api = mocker.patch('fluidreview.api.FluidReviewAPI')
+    mock_api().get.return_value.json.return_value = {
+        'id': award_id,
+        'name': award_name,
+        'tag_line': 'The very best!',
+        'description': '',
+        'amount_to_pay': '450',
+    }
+    data = {
+        'date_of_birth': '',
+        'user_email': 'veteran-grants-9463shC',
+        'amount_to_pay': amount_to_pay,
+        'user_id': user_id,
+        'submission_id': 4533767,
+        'award_id': award_id,
+        'award_cost': award_cost,
+        'award_name': 'TEST CAMP'
+    }
+    body = json.dumps(data)
+    hook = WebhookRequest(body=body)
+
+    with patch(
+        'fluidreview.api.MailgunClient.send_individual_email',
+    ) as send_email:
```

Comment:
Could also do `send_email = mocker.patch('fluidreview.api.MailgunClient.send_individual_email')` instead, near beginning of test code.